iml-manager-cli/src/manager_cli_error.rs

```diff
@@ -2,14 +2,45 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-pub type Result<T> = std::result::Result<T, ImlManagerCliError>;
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Do we need to derive `Clone`, `PartialEq`, `Eq`?

Reply:
Nah, i'll remove them.